wagtail/wagtailadmin/templates/wagtailadmin/shared/menu_item.html

```diff
@@ -0,0 +1,3 @@
+<li class="menu-item menu-{{name}}{% if active %} menu-active{% endif %}">
```

Comment:
A minor formatting quibble: we generally have a space either side of the variable double-braces: `{{ name }}` rather than `{{name}}`